### PR TITLE
Update Rule “make-newcomers-feel-welcome/rule”

### PR DESCRIPTION
**Pull request by camillars**
Automatically generated by Netlify CMS

### CODE REVIEW

File: rules/make-newcomers-feel-welcome/rule.md [hunk 1]
```diff
@@ -1,61 +1,69 @@
...
-
-There are few simple ways to help new hires feel more comfortable as they adjust to their new workspace:
+There are a few simple ways to help new hires feel more comfortable as they adjust to their new workspace:
 
 <!--endintro-->
 
 ### Before they start
 
-Prior to the newcomer’s arrival, make sure their desk and work area are clean by giving the desktop, monitor, mouse and keyboard a wipe with a damp cloth. Then test that the equipment on their desk, such as the computer, phone and any other electronics, is in good working condition.
+**Welcome on the day** - Make sure a team member will be there to greet them on the day!
```
Comment (wicksipedia) [inline rules/make-newcomers-feel-welcome/rule.md:25]: Maybe change from:
> **Welcome on the day**

to

> **Plan to welcome on the day**

Comment (camillars) [reply rules/make-newcomers-feel-welcome/rule.md]: Done

**Review by wicksipedia — APPROVED** [review]
lgtm

**Review by AshAnil — APPROVED** [review]
Good work 👍

**Pull request merged**